rust/utils/abigen/src/lib.rs

```diff
@@ -1,12 +1,20 @@
-use fuels_code_gen::ProgramType;
 use std::collections::BTreeSet;
 use std::ffi::OsStr;
 use std::fs::{self, File};
 use std::io::Write;
 use std::path::Path;
 
+// use fuels_code_gen::ProgramType;
 use inflector::Inflector;
 
+// to avoid "use of undeclared crate or module `fuels_code_gen`" error
```

Comment:
You should avoid copying in dependency code as much as possible, there's probably a root cause for the error that needs to be fixed instead. Can you share steps to reproduce the problem?